modules/zone/zone.go

```diff
@@ -189,52 +156,44 @@ func (s *RoutineLookupFactory) MakeLookup() (zdns.Lookup, error) {
 //
 type GlobalLookupFactory struct {
 	zdns.BaseGlobalLookupFactory
-	Glue        *map[string][]string
-	GlueLock    sync.RWMutex
-	Subfactory  alookup.GlobalLookupFactory
-	CacheFactor int
-	CacheHash   *cachehash.CacheHash
-	CHmu        sync.Mutex
-	Manager     CallbackManager
+	Glue         *map[string][]string
+	GlueLock     sync.Mutex
+	Subfactory   alookup.GlobalLookupFactory
+	CacheFactor  int
+	GlueFilePath string
+	Hash         *cachehash.CacheHash
```

Comment:
What is this a Hash of? What is it used for? Generally the type of a variable is not a good name for it.